cloudbuild.yaml

```diff
@@ -5,11 +5,12 @@ options:
   substitution_option: ALLOW_LOOSE
   machineType: N1_HIGHCPU_8
 steps:
-  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20211118-2f2d816b90
+  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20220506-774f302a94
     entrypoint: bash
     env:
       - TAG=$_GIT_TAG
       - HOME=/root  # for docker buildx
+      - GOOGLE_SERVICE_ACCOUNT_NAME=448637284062@cloudbuild.gserviceaccount.com
```

Comment:
Mind adding a comment on where this service account name comes from?

Reply:
Done